### DISCUSSION TITLE
Decimal instructions

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#363] [Q&A]
With the existing decimal functions, are we complete? 
- I added I2DEC for a conversion from an Integer.
- There is no decimal from Float function in the library, how do we handle it?
-  do we need a Decimal to integer/float conversion?
- There is no DEC compare function implemented, I guess we need something there. 
- if so, I'd plea for a very basic one  DCLC r1,r2,r3, r1 is -1 for LT, 0 for equal, and 1 for greater. LE, GE, and NE, can be implemented by rexx code following the compare instruction.

**Comment by adesutherland** [comment]
Just to remind us in the future, of what we agreed yesterday!

- I added I2DEC for a conversion from an Integer - **For the compiler's sake we need to use the standard naming (e.g. itod)**
- There is no decimal from Float function in the library, how do we handle it? **Noted - we will have to investigate options**
- do we need a Decimal to integer/float conversion? **Yes**
- There is no DEC compare function implemented, I guess we need something there. **Yes and they will have to follow the pattern - e.g. deq, dne, dgt**
